Makefile

```diff
@@ -307,6 +307,10 @@ unit-test-go-coverpkg: $(GOTESTSUM)
 lint-go:
 	$(GOLANGCI_LINT) run --verbose
 
+.PHONY: lint-go-fix
+lint-go-fix:
```

Comment:
Can we spin off this lint-go-fix target (and generated results) into their own PR? It's a lot of changes to review that go beyond just updating the Go version.

Reply:
Yes, but please don't review it. Not only can you not change the output but it's auto-generated based on the lint rules.